apps/grpc/single/cmd/run.go

```diff
@@ -69,6 +69,10 @@ The execution client must implement the Evolve execution gRPC interface.`,
 			return err
 		}
 
+		if genesis.DAStartHeight == 0 && !nodeConfig.Node.Aggregator {
+			logger.Warn().Msg("da_start_height is not set in genesis.json, ask your chain developer")
```

Comment:
is it okay for this to just be a warning?

Reply:
Yes, this is in case node operators and sequencers all start beforehand, but as well as a reminder for node operators to ask the chain dev for an update.